src/weekly/query.js

```diff
@@ -46,9 +45,10 @@ async function queryZuora (deliveryDate, config: Config) {
          ProductRatePlanCharge.ProductType__c = 'Guardian Weekly' AND
          RatePlanCharge.EffectiveStartDate <= '${formattedDate}' AND
          (
-          (Subscription.AutoRenew = true AND RatePlanCharge.EffectiveEndDate >= '${formattedDate}') OR
-          (Subscription.AutoRenew = false AND Subscription.TermEndDate >= '${formattedDate}')
-         )`
+          (Subscription.AutoRenew = true AND RatePlanCharge.EffectiveEndDate > '${formattedDate}') OR
+          (Subscription.AutoRenew = false AND Subscription.TermEndDate > '${formattedDate}')
+         ) AND 
+         ( RatePlan.AmendmentType != 'RemoveProduct' OR RatePlanCharge.EffectiveEndDate >= '${formattedDate}' )`
```

Comment:
your previous condition included the case where dates were equal so technically  if you are negating this then it should not include this case

Reply:
I'm going to need to think about this, as it does change the output and I'm not sure what's right here.

Reply:
I think this will work:

```
( RatePlan.AmendmentType is null OR RatePlan.AmendmentType != 'RemoveProduct' OR RatePlanCharge.EffectiveEndDate > '${formattedDate}' )
```